# this string allows to change the path to the stored recognition models
# string
# data_directory: path_to_data -> please change this parameter in the launch file

# this flag enables or disables the face recognition step
# bool
enable_face_recognition: true

# this list specifies the persons who shall be recognized by the face recognizer, comment out this line to avoid any specification at startup
# string[] e.g. ['albert', 'bernhard', 'caesar']
#identification_labels_to_recognize: ['albert', 'bernhard', 'caesar']

# desired width and height of the eigenfaces (=eigenvectors)
# int
eigenface_size: 100

# number of eigenvectors per person to identify -> controls the total number of eigenvectors
# int
eigenvectors_per_person: 5

# threshold to facespace, detections with a higher score are not considered as being faces
# double
threshold_facespace: 22000

# threshold to detect unknown faces, detections with a higher score receive the unknown label and not a name
# double
threshold_unknown: 4000

# metric for nearest neighbor search in face space: 0 = Euklidean, 1 = Mahalanobis, 2 = Mahalanobis Cosine
# int
metric: 0

# enables some debug outputs
# bool
debug: true
